Eatery/Onboarding/Controllers/OnboardingInfoViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  OnboardingInfoViewController.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/15/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+class OnboardingInfoViewController: OnboardingViewController {
+
+    private let stackView = UIStackView()
+    private let imageView = UIImageView()
+    private let nextButton = UIButton()
+    private var onboardingImage = UIImage()
+
+    init(title: String, subtitle: String, image: UIImage!) {
```

Comment:
Change `image: UIImage?`. I think making it a normal optional is fine. If the image that's passed in is nil then `imageView` can display nothing.

Reply:
Why do you have `UIImage!`? Is the image optional? In that case say `UIImage?`. Also, you can remove the assignment on line 16, make it a `let` constant, and move `self.onboardingImage = image` to line 18 right before the super call. 

That is, line 16 would be `private let onboardingImage: UIImage`, and you move line 20 before line 19.